ui/app/models/config-ui/message.js

```diff
@@ -2,21 +2,96 @@
  * Copyright (c) HashiCorp, Inc.
  * SPDX-License-Identifier: BUSL-1.1
  */
-
 import Model, { attr } from '@ember-data/model';
 import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
-import { isAfter } from 'date-fns';
+import { isAfter, format, addDays, startOfDay } from 'date-fns';
 import { parseAPITimestamp } from 'core/utils/date-formatters';
+import { withModelValidations } from 'vault/decorators/model-validations';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+
+export const localDateTimeString = "yyyy-MM-dd'T'HH:mm";
```

Comment:
I think this could be useful to either add to the model (since it looks like it's only referenced in the context of the model) or add to `core/utils/date-formatters` since this will be useful anytime we want to format a date for the date local html input: i.e. `datetimeLocalStringFormat` (i made `time` lowercase since MDN docs have datetime as one word)